internal/cephfs/controllerserver.go

```diff
@@ -815,6 +815,18 @@ func (cs *ControllerServer) CreateSnapshot(
 			}
 		}
 
+		// Update snapshot-name/snapshot-namespace/snapshotcontent-name details on
+		// sunvolume snapshot as metadata on restart of provisioner pod when
```

Comment:
no need to mention restart just mention updating metadata in case a snapshot already exists.